tools/wpt/browser.py

```diff
@@ -431,8 +431,52 @@ class Servo(Browser):
     product = "servo"
     requirements = "requirements_servo.txt"
 
-    def install(self, dest=None):
-        raise NotImplementedError
+    def platform_string(self):
+        platform = {
+            "Linux": "linux",
+            "Windows": "win",
+            "Darwin": "mac"
+        }.get(uname[0])
+
+        if platform is None:
+            raise ValueError("Unable to construct a valid Servo package for current platform")
+
+        if platform == "linux":
+            bits = "-%s" % uname[4]
+            extension = ".tar.gz"
+        elif platform == "win":
+            bits = "64" if uname[4] == "x86_64" else "32"
+            extension = ".msi"
+        else:
+            bits = ""
+            extension = ".dmg"
```

Comment:
All of this is unused; I'd suggest returning a tuple (`return (platform, extension)`), though, so you don't need two methods.

Reply:
Did you test the new code? I don't see how it can work, because you return a single string instead of a tuple.

Reply:
I am having trouble in understanding how to test my changes locally. Could you please guide me on that? @Ms2ger

Reply:
There is still a bunch of unused code in this method. I agree that combining the platform and extension methods together makes sense.